File: app/src/main - 副本/java/com/zmcursor/mydaliy/UI/main/MainPresenter.java
package com.zmcursor.mydaliy.UI.main;

import com.zmcursor.mydaliy.Model.Bean.StoriesOfDate;
import com.zmcursor.mydaliy.Model.Bean.TopStory;

import java.util.List;

/**
 * Created by ZMcursor on 2018/6/2 0002.
 */

public interface MainPresenter {

//    void initData();

    void refresh();

    void loadMore();

    void saveDate();

    List<TopStory> getTopStories();

    List<StoriesOfDate> getStoriesOfDateList();

}

File: app/src/main/java/com/zmcursor/mydaliy/UI/main/MainPresenter.kt
package com.zmcursor.mydaliy.UI.main

import com.zmcursor.mydaliy.Model.Bean.StoriesOfDate
import com.zmcursor.mydaliy.Model.Bean.TopStory

/**
 * Created by ZMcursor on 2018/6/2 0002.
 */

interface MainPresenter {

    val topStories: List<TopStory>

    val storiesOfDateList: List<StoriesOfDate>

    //    void initData();

    fun refresh()

    fun loadMore()

    fun saveDate()

}
